src/main/java/ua/su/controllers/MedicalProcedureController.java

```diff
@@ -1,4 +1,47 @@
 package ua.su.controllers;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import ua.su.domain.MedicalProcedure;
+import ua.su.service.MedicalProcedureService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
 public class MedicalProcedureController {
-}
+
+    private final MedicalProcedureService medicalProcedureService;
+
+    public MedicalProcedureController(MedicalProcedureService medicalProcedureService) {
```

Comment:
It is not a bug but pay attention that you can avoid mark constructor as @Autowired, but it should be only one so in this case spring creates a bean using this constructor. Also to make code consistency I would add @Autowired here

Reply:
added the Autowired annotation to the constuctor.